examples/mobilenet/README.md

```diff
@@ -0,0 +1,13 @@
+# MobileNet
+
+For evaluation, please go to [`examples/classification`](https://github.com/chainer/chainercv/tree/master/examples/classification).
+
+## Convert Caffe model
+Convert TensorFlow's `*.ckpt` to `*.npz`.
+
+```
+$ python caffe2npz.py mobilenetv2 <source>.ckpt <target>.npz
```

Comment:
`tf2npz.py`?

Reply:
Ah, you are using``tfckpt2npz.py` as the name of conversion script.

Reply:
For the consistency with `DeepLab`, how about renaming it `tf2npz.py` ?
https://github.com/chainer/chainercv/blob/master/examples/deeplab/tf2npz.py

Reply:
DeepLab's tf2npz.py uses tensorflow's frozen model, but this tfckpt2npz uses tf's checkpoint. Frozen model and checkpoint are different. Frozen model is removed some BNs so not to be used for training.

Reply:
OK